src/java.desktop/share/classes/sun/print/PathGraphics.java

```diff
@@ -1142,6 +1146,17 @@ protected BufferedImage getBufferedImage(Image img) {
         }
     }
 
+    public static BufferedImage convertToBufferedImage(MultiResolutionImage multiResolutionImage,
+                                                       double width, double height ) {
+        Image resolutionImage = multiResolutionImage.getResolutionVariant(width, height);
+        BufferedImage bufferedImage = new BufferedImage(resolutionImage.getWidth(null),
+                                                        resolutionImage.getHeight(null),
+                                                        BufferedImage.TYPE_INT_ARGB);
+        Graphics2D g2d = bufferedImage.createGraphics();
```

Comment:
I'm not sure whether we have to dispose it here at this context.

Reply:
Since you are creating it locally, I guess you can dispose it.

Reply:
I'm sure whether a new Graphics2D object will be created or reference will be passed, just vaguely remember a conversation regarding disposing it and suggested not to dispose since a reference will be passed on. So not sure should we actually dispose here or not. And also I see in most of the places its not been disposed.

Reply:
I guess you mentioned about this thread https://github.com/openjdk/jdk/pull/17609#discussion_r1483263329.
But here the graphics object is passed as parameter.

Reply:
Yeah right. Thank you for clarification.